src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -468,20 +468,23 @@
           display: block;
           position: absolute;
           width: calc(0.5rem + #{$clr-rem-1px});
-          right: -0.5rem;
+          right: -0.25rem;
           top: -0.25rem;
           cursor: col-resize;
           height: calc(100% + 0.5rem - #{$clr-default-borderwidth});
           z-index: map-get($clr-layers, datagrid-header);
         }
-        .datagrid-column-handle-tracker {
+        .datagrid-column-resize-tracker {
```

Comment:
Isn't this a breaking change?

Reply:
This is an element that's in our template. We tell people not to mess with the styles of internal datagrid stuff, so I don't consider this a breaking change.